src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -75,7 +93,12 @@ public Double averageSumOdd(List<Integer> numbers) {
      **/
     public List<People> workablePeople(int fromAge, int femaleToAge,
                                        int maleToAge, List<People> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(people -> people.getSex() == People.Sex.MAN
+                        && people.getAge() >= fromAge && people.getAge() <= maleToAge
```

Comment:
`people.getAge() >= fromAge` is common to male and female.
try to evoid code duplication